policyengine_us/tests/policy/baseline/gov/states/oh/tax/income/credits/oh_non_public_school_credits.yaml

```
@@ -0,0 +1,44 @@
+- name: AGI Not eligible
+  period: 2021
+  input:
+    oh_agi: 100_000
+    non_public_school_tuition: 1000
+    state_code: OH
+  output:
+    oh_non_public_school_credits: 0
+
+- name: No non public tuition
+  period: 2021
+  input:
+    oh_agi: 80_000
+    non_public_school_tuition: 0
+    state_code: OH
+  output:
+    oh_non_public_school_credits: 0
+
+- name: AGI less that 50_000
```

Only need underscores in values
```suggestion
- name: AGI less than $50,000
```